source/simulator/solver.cc

```diff
@@ -458,7 +458,15 @@ namespace aspect
   }
 
 
-
+  dealii::SolverControl::State
+  solver_callback (const unsigned int iteration,
```

Comment:
Put the function in an anonymous namespace. No need to exert the linker more than necessary.